plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -106,17 +531,31 @@ const storeDetails = {
 		},
 		expectedNumberOfIndustries: 8, // There are 8 checkboxes on the page (in Liberia), adjust this constant if we change that
 		industries: {
-			other: 'Other',
+			other: getTextForLanguage()['Other'],
 		},
 		products: {
-			physical: 'Physical products',
-			downloadable: 'Downloads',
+			physical: getTextForLanguage()['Physicalproducts'],
+			downloadable: getTextForLanguage()['Downloads'],
 		},
 	},
 };
 
+function getTextForLanguage(langCode = 'en') {
+	if (typeof LANGUAGE !== 'undefined') {
```

Comment:
Would it be ok to shorten this to `if ( LANGUAGE ) {`? `if` would get `true` when `LANGUAGE` is defined.